app/styles/app.scss

```diff
@@ -15,42 +15,45 @@
  * ELEMENTS: Plain html tag styling
  * OBJECTS: OOCSS unstyled objects (namespace: '.au-o-...')
  * COMPONENTS: Components (namespace: '.au-c-...')
+ * PLUGINS: Third-party styles
  * UTILITIES: Single function helper classes (namespace: '.au-u-...')
  */
 
 
-// VARIABLES
-@import "appuniversum/s-colors.scss";
-@import "appuniversum/s-global";
-@import "appuniversum/s-utilities";
+// SETTINGS
+@import "ember-appuniversum/s-colors.scss";
+@import "ember-appuniversum/s-global";
+@import "ember-appuniversum/s-utilities";
+@import "ember-appuniversum/s-root";
 
 
-// MIXINS
-@import "appuniversum/t-font-size";
-@import "appuniversum/t-sass-mq";
+// TOOLS
+@import "ember-appuniversum/t-font-size";
+@import "ember-appuniversum/t-sass-mq";
 
 
 // GENERIC
-@import "appuniversum/g-reset";
-@import "appuniversum/g-box-sizing";
-@import "appuniversum/g-font";
+@import "ember-appuniversum/g-reset";
+@import "ember-appuniversum/g-box-sizing";
+@import "ember-appuniversum/g-font";
 
 
 // ELEMENTS
-@import "appuniversum/e-page";
+@import "ember-appuniversum/e-page";
 
 
 // OBJECTS
-@import "appuniversum/o-box";
-@import "appuniversum/o-flow";
-@import "appuniversum/o-grid";
-@import "appuniversum/o-layout";
-@import "appuniversum/o-region";
+@import "ember-appuniversum/o-box";
+@import "ember-appuniversum/o-flow";
+@import "ember-appuniversum/o-grid";
+@import "ember-appuniversum/o-layout";
+@import "ember-appuniversum/o-region";
 
 
 // COMPONENTS
```

Comment:
I would suggest to load the separate sections as groups. Makes it a lot easer to update ember-appuniversum in the future.

E.g
```
@import "ember-appuniversum/a-settings";
@import "ember-appuniversum/a-tools";
@import "ember-appuniversum/a-generic";
@import "ember-appuniversum/a-elements";
@import "ember-appuniversum/a-objects";
@import "ember-appuniversum/a-components";
@import "ember-appuniversum/a-plugins";
@import "ember-appuniversum/a-utilities";
```